src/Mod/TechDraw/Gui/DlgPrefsTechDraw.ui

```diff
@@ -156,6 +168,9 @@
         </item>
         <item row="5" column="1">
          <widget class="Gui::PrefDoubleSpinBox" name="dsb_TemplateDot">
+          <property name="toolTip">
+           <string>EditableText Dot Size in mm</string>
```

Comment:
Line [610](https://github.com/FreeCAD/FreeCAD/pull/1182/files#diff-6624f017955d1c4368025bf16d1e6ea5R610) displays the word `millimeter` should this line just display it as well ?